src/integration-tests/introspector/README

```diff
@@ -19,8 +19,11 @@ Usage:
     One way to do this when running locally is to run the integration tests.
     (Note that this test doesn't depend on Operator image as of 2018/10/16, but may in the future.)
 (2) Optionally specify values for input env vars (see introspectTest.sh for the list).
-(3) Run introspectTest.sh
-
+(3) For non Model In Image domain type.
+        Run introspectTest.sh
+    For Model In Image domain type.
+    export DOMAIN_SOURCE_TYPE=FromModel
+    Run introspectTest.sh
```

Comment:
Indent lines 25 and 26 four more spaces.  Right now they line up right under 'For Model'.   Change 'type.' to 'type:'.   Change 'For ' to 'For a '.

Reply:
done